src/index.ts

```diff
@@ -7,3 +8,5 @@ export const Code = {
     context,
     utils,
 };
+
+export { Types };
```

Comment:
What happened here?

Reply:
Probably leftovers of solving local issues